proto/storage/vulnerability.proto

```diff
@@ -58,6 +57,7 @@ message EmbeddedVulnerability {
   repeated CVSSScore cvss_metrics = 21;
   float nvd_cvss = 22; // @gotags: search:"NVD CVSS,store"
   EPSS epss = 23;
+  string advisory = 24;
```

Comment:
Curious, why did you move this?  I expected no changes in this file as scanner already made these changes.

Reply:
@dashrews78 Both CWF and scanner made changes and scanner merged their changes first